util/maven/generate_poms.py

```diff
@@ -34,14 +34,14 @@ def deps_of(label):
 def exports_for(label):
   return _shell('bazel query "labels(exports, %s)"' % label)
 
-def pom_deps(label):
+def pom_deps(label, include_exports = True):
   accumulated_deps = set()
-  for dep in deps_of(label):
+  for dep in deps_of(label) + (exports_for(label) if include_exports else []):
     if dep.startswith("@local_jdk//:"): continue
     if dep.startswith(('//:', '//third_party:')):
       for export in exports_for(dep):
         accumulated_deps.add(export)
-        accumulated_deps.update(pom_deps(export))
+        accumulated_deps.update(pom_deps(export, False))
```

Comment:
The benefit of putting the classifier in a tag is that these can happen transitively